developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst

```diff
@@ -57,9 +69,50 @@ App container parameters with pascal case names ``AppName``, ``UserId`` and ``We
+* Middlewares can be registered globally (see :ref:`global_middlewares`, `PR#36310 <https://github.com/nextcloud/server/pull/36310>`_ by `ChristophWurst <https://github.com/ChristophWurst>`_)
 
 Removed APIs
 ^^^^^^^^^^^^
 
-tbd
\ No newline at end of file
+* ``OCP\BackgroundJob\IJobList::getAll`` method was removed (`PR#36073 <https://github.com/nextcloud/server/pull/36073>`_ by `come-nc <https://github.com/come-nc>`_)
+* 3rdparty dependency ``php-ds/php-ds`` was removed (`PR#36198 <https://github.com/nextcloud/server/pull/36198>`_ by `kesselb <https://github.com/kesselb>`_)
+* ``OCP\Contacts\IManager::getAddressBooks`` method was removed (`PR#34329 <https://github.com/nextcloud/server/pull/34329>`_ by `come-nc <https://github.com/come-nc>`_)
+* ``OCP\Util`` loglevel constants were removed (`PR#34329 <https://github.com/nextcloud/server/pull/34329>`_ by `come-nc <https://github.com/come-nc>`_)
+* 3rdparty dependency ``nikic/php-parser`` was removed (`PR#36393 <https://github.com/nextcloud/server/pull/36393>`_ by `kesselb <https://github.com/kesselb>`_)
+* Deprecated ``OCP\AppFramework\Db\Mapper`` was removed. You can easily migrate to ``OCP\AppFramework\Db\QBMapper``, which is doing the same thing using the query builder instead of string based queries. (`PR#34490 <https://github.com/nextcloud/server/pull/34490>`_ by `rullzer <https://github.com/rullzer>`_)
+* Deprecated classes of ``OCP\Dashboard`` were removed (`PR#35966 <https://github.com/nextcloud/server/pull/35966>`_ by `juliushaertl <https://github.com/juliushaertl>`_)
+
+Added APIs
+^^^^^^^^^^
+
+* New ``OCP\Authentication\Token\IProvider`` for authentication providers: Created a new public interface ``OCP\Authentication\Token\IProvider`` with a method invalidateTokensOfUser to invalidate all tokens of a specific user. ``OC\Authentication\Token\Manager`` implements ``OCP\Authentication\Token\IProvider``. (`PR#36033 <https://github.com/nextcloud/server/pull/36033>`_ by `individual-it <https://github.com/individual-it>`_)
+* ``Auto-Submitted`` header for emails: There now is a new method on the ``OCP\Mail\IMessage`` interface ``IMessage::setAutoSubmitted()``. With this method you can specify that an email was an automatic email or response, to allow mail servers to better detect if an out-of-office reply should be sent, better store/filter the emails and so on. Possible values are documented in the ``OCP\Mail\Headers\AutoSubmitted`` interface. (`PR#36033 <https://github.com/nextcloud/server/pull/36033>`_ by `bennet0496 <https://github.com/bennet0496>`_)
+* ``OCP\BackgroundJob\IJobList::getJobsIterator`` method was added (`PR#36073 <https://github.com/nextcloud/server/pull/36073>`_)
+* New ``OCP\BeforeSabrePubliclyLoadedEvent`` event dispatched on public webdav endpoints (it can be used just like ``OCP\SabrePluginEvent`` to inject additional Sabre plugins in apps for example) (`PR#35789 <https://github.com/nextcloud/server/pull/35789>`_)
```

Comment:
documentation on the events page is missing

Reply:
exactly :)

Reply:
thanks, we will do it today. _to clarify: we received task only to update one page, so we did what we were asked, didn't know that we should update more info_ :)